### PR TITLE
Add support for context of input and profile sets

### PR DESCRIPTION
**Pull request by jelovirt**
Add support for input and profile set context. This abstraction is not strictly required by publishing engine, but is a logical unit for editors and CMSs.

Signed-off-by: Jarno Elovirta <jarno@elovirta.com>

### CODE REVIEW

File: src/main/java/org/dita/dost/invoker/Main.java [hunk 1]
```diff
@@ -556,8 +556,6 @@ public Main() {
      * @since Ant 1.6
      */
     private void processArgs(final String[] arguments) {
-        String searchForThis = null;
```
Comment (jelovirt) [inline src/main/java/org/dita/dost/invoker/Main.java]: Removed unused features of `Main` in the same PR.

**Comment by jelovirt** [comment]
`inputs` is a wrapper for multiple input files, to be implemented.

`<profile>` element is used in XML serialization, not `<profiles>`

**Comment by jelovirt** [comment]
We can remove `<inputs>` and allow e.g.

```xml
<context>
  <input href="first.topic"/>
  <input href="second.topic"/>
  <profile>
    <ditaval href="eu.ditaval"/>
    <ditaval href="beginner.ditaval"/>
  </profile>
</context>
```

I think the format of the serialization should be deferred to a later PR.

**Comment by raducoravu** [comment]
> `<inputs>` is a wrapper for multiple input files, to be implemented.

I still do not understand what this means. So if I refer two DITA Maps do I get in the output a single PDF containing the merged content or two PDFs, one for each map?

**Comment by jelovirt** [comment]
One output per `<deliverable>`.

**Comment by raducoravu** [comment]
Ok, understood.

**Pull request merged**